src/pagination.rs

```diff
@@ -34,35 +35,32 @@ fn extract_page_path_from(p: &liquid::Object) -> String {
     .into_owned()
 }
 
-pub(crate) fn extract_page_path(p: &liquid::Object, config: &PaginationCfg) -> PathBuf {
-  PathBuf::from(match config.include.as_str() {
-    "all" => extract_page_path_from(&p),
+pub fn extract_page_path(p: &liquid::Object, config: &PaginationConfig) -> PathBuf {
+  PathBuf::from(match config.include {
+    Include::All => extract_page_path_from(&p),
     _ => "".to_owned(),
```

Comment:
Please cover all cases.  Its a big help for when you add new ones, the compiler will tell you when you miss something.

Also, `""` or `panic`/`unreachable`?